### PR TITLE
Removes cat ears from the autodrobe vendor and loadout

### PR DESCRIPTION
**Pull request by FreeStylaLT**
## What Does This PR Do
Removes cat ears from the Autodrobe vendor and its supply packs.

## Why It's Good For The Game
Reduces the amount of overall unpleasant roleplay that comes from an amount of players that decide to don it.

Cat ears would still be accessible by other, less publicly available means.

## Changelog
:cl:FreeStylaLT
del: Removed cat ears from Autodrobe and loadout
/:cl:

### CODE REVIEW

**Comment by MRgaty** [comment]
Yas

**Comment by Wotzizname** [comment]
No,they would add a catgirl race then

**Comment by Mitchs98** [comment]
Why, though? I see no reason to remove them, personally. Probably a bit biased since I have two characters that use them. Removing them from the autodrobe is fair enough, I guess, but removing them from loadout too? Forcing people to have to buy some dumb crate over and over for a small chance to get them? I definitely disagree. 

They aren't really hurting anyone at all just because some people dislike them. Some people dislike just about everything, yet it's not removed or changed.

**Comment by Carthusia** [comment]
I also see no real reason to remove an item of player choice that is entirely cosmetic? 

Can you please define what you mean by "unpleasant roleplay that comes from an amount of players that decide to don it."

and how their RP style is any less valid than everyone else's?

**Comment by Birdtalon** [comment]
A thousand slime girls squishing in rage.

**Comment by Spacemanspark** [comment]
> Can you please define what you mean by "unpleasant roleplay that comes from an amount of players that decide to don it."

and how their RP style is any less valid than everyone else's?

The majority of users with this item are slimepeople that sit in front of the bridge spamming *squish and emoting about nyaing. This isn't an exaggeration.
While normally I'd agree, especially with a cosmetic item, this one item seems to be giving people more excuse to be an anime ripoff stereotype.
While we don't expect people to put in hours upon hours developing characters, or even half an hour, we want more effort than 'kawaii person with cat ears, nyaaaaaaaaa~' for a backstory.
Put simply, if this is your style of roleplay... you might want to try a different server.
If you want cat ears, we have the tajara race.

I do apologize if this seems harsh to some, but it is an issue.

**Comment by owenowen101** [comment]
Paranoia and spacemen not anime and catgirls OK

**Comment by RaidarV** [comment]
Haha, good meme, no. I will have my pink-maid-catgirl-lizard-man.

**Comment by MRgaty** [comment]
Most of the players I see using cat ears focus their whole character in them, nyaaaing, with weird names and being usually uneasy to interact with.
If this removes at least some of those characters and encourages better characters than cute uwu anime girls, then is a victory.

**Comment by Carthusia** [comment]
If it is this much of an issue "I personally never notice it being one, to each their own" perhaps a system of Karmalocked clothing could be considered? It doesn't prevent it from happening though adds a level of "You need to be at least this trusted by the community to use it"

Other various bits of clothing could be added to that normally require deep dive searching onboard the station, such as the mailman outfit "A slightly different blue jumpsuit" and various vox civilian clothing in the code, but not actually available for usage, just examples but obviously more could be added.

**Comment by MRgaty** [comment]
> If it is this much of an issue "I personally never notice it being one, to each their own" perhaps a system of Karmalocked clothing could be considered? It doesn't prevent it from happening though adds a level of "You need to be at least this trusted by the community to use it"
> 
> Other various bits of clothing could be added to that normally require deep dive searching onboard the station, such as the mailman outfit "A slightly different blue jumpsuit" and various vox civilian clothing in the code, but not actually available for usage, just examples but obviously more could be added.

Normally people with characters like those are not newbies, some of them even have more than 3k hours

**Comment by Citinited** [comment]
I see no problem with this

**Comment by Carthusia** [comment]
> Normally people with characters like those are not newbies, some of them even have more than 3k hours

If that's the case, then simply removing the item in question isn't going to change their characters, nothing you could do will ever change how they act.

Again, I personally never notice this being an issue that is of such severity that it actually causes me to dislike the round or my experience because of it.

Considering the above: Will this really amount to anything?

**Comment by Spacemanspark** [comment]
It at least doesn't blatantly encourage the behavior.

**Comment by Carthusia** [comment]
> It at least doesn't blatantly encourage the behavior.

Perhaps, though cat ears or not nothing will stop that kind of behavior short if flat out banning it within the rules, which is a step WAY to far in my eyes.

**Comment by AzuleUtama** [comment]
Echoing what Gaty is saying above, this isn't the case of some newbie players vending the item and going around acting in a dumb way - but instead people who've played on the server for a much longer period of time having it set to a loadout option and just running with it as their gimmick for the round.

Honestly, just removing it as a loadout option would be enough - at least then it limits the number you can get at round start. Still, won't argue with it being removed entirely since the whole catgirl clique just doesn't fit a game like this - especially with us trying to be MRP.

**Comment by Spacemanspark** [comment]
> Perhaps, though cat ears or not nothing will stop that kind of behavior short if flat out banning it within the rules, which is a step WAY to far in my eyes.

Why is this too far, exactly? That would simply be enforcing our standards.
Again... if that's your style of roleplaying, you may very well find yourself much happier on another server that caters towards that. We really _aren't_ that type of server.

**Comment by thatdanguy23** [comment]
Praise Space Jesus, we might actually get higher roleplay that amounts to more than just trashy, repetitive meme characters now.

**Comment by Carthusia** [comment]
> Why is this too far, exactly? That would simply be enforcing our standards.
> Again... if that's your style of roleplaying, you may very well find yourself much happier on another server that caters towards that. We really _aren't_ that type of server.

It's not that its my style of RP (Never will be) but that its the idea of removing player choice that I dislike. 

If someone wants to play a weeb, does it really hurt anyone?

**Comment by Spacemanspark** [comment]
Honestly, yes, as it drags down the atmosphere and goes against the standards set here.
You're at a workplace, not an anime convention. Some (maybe more than some) silliness is fine, obviously, and encouraged! But there's a limit.
You can play a weeb, frankly... just don't be so over the top?

**Comment by Quantum-M** [comment]
> It's not that its my style of RP (Never will be) but that its the idea of removing player choice that I dislike.

There aren't that many choices when it comes to the cat ears. The most of the times I seen it being used was for either creepy cat girl memes, or as a joke when trying to put them on someone.

And as stated, it usually lowers the roleplay atmosphere a bit when people wears them all the place (and I think the tail it comes with the ears do not help either).

**Comment by Carthusia** [comment]
> Honestly, yes, as it drags down the atmosphere and goes against the standards set here.
> You're at a workplace, not an anime convention. Some (maybe more than some) silliness is fine, obviously, and encouraged! But there's a limit.
> You can play a weeb, frankly... just don't be so over the top?

Cant deny it,  you will miss seeing an over the top cat person getting beat up, or, and i can't remember their name but you'll probably be able to ID them just by this description: The naked 1 armed, no legged fat guy in a wheelchair with a tail.

I will admit, that is one character where every single time I see them, i do cringe, and not one vox in the surrounding area when he was hurt, or fell out of his chair, ever wanted to touch him as he was to gross.

**Comment by MRgaty** [comment]
> > Honestly, yes, as it drags down the atmosphere and goes against the standards set here.
> > You're at a workplace, not an anime convention. Some (maybe more than some) silliness is fine, obviously, and encouraged! But there's a limit.
> > You can play a weeb, frankly... just don't be so over the top?
> 
> Cant deny it, you will miss seeing an over the top cat person getting beat up, or, and i can't remember their name but you'll probably be able to ID them just by this description: The naked 1 armed, no legged fat guy in a wheelchair with a tail.
> 
> I will admit, that is one character where every single time I see them, i do cringe, and not one vox in the surrounding area when he was hurt, or fell out of his chair, ever wanted to touch him as he was to gross.

To be fair if you want to see a fat catgirl being beaten there is no better place than hippie.

**Comment by Darkmight9** [comment]
Husky bigins is the guy you're thinking of Car. No idea why he wears the cat ears but it's likely to add more 5o his grossness factor.

**Comment by Fox-McCloud** [comment]
Regardless of if this goes through or support/lack of support....

Pretending like this is somehow going to improve RP, make things better, or have a sizeable impacts is kidding yourself.

Individuals that are attracted to a particular type of, well, to put it bluntly "snowflakey" playstyle/attention grabby playstyle are just going to find other ways.

If it's not cat ears, it'll be the maid outfit. If it's not the maid outfit, it'll be wearing bright/weird fur/slime/body colors. If it's not that, it'll be the most ridiculous description ever--if not that, it'll be making the most obnoxious personality type and mannerisms you can muster. Point is; there's a thousands outlets for this.

**Comment by Shockpoint** [comment]
Why are we trying to defend this with our expected level of roleplay?
Often have we seen people buckled to beds (usually Ian's bed) with cat ears and the fluffy cuffs if they get ahold of those. That's disgusting, I don't want to see that. I shouldn't have to bwoink everyone who does this.

Simply put, shit like this needs to go. It promotes less than adequate roleplay, implies we're fine with borderline ERP/weird shit, and honestly just does not fit the server as a whole.

**Comment by Ty-Omaha** [comment]
Good.

There's absolutely 0 reason to have slimes standing infront of the bridge with cat ears going *squish the entire round with their friend they are dragging around on a MRP server.

**Comment by Darkmight9** [comment]
Everyone is focusing on the memey and lewd cat ears users and yet no one is thinking about the people that just use them because it looks cute on their character. Yes I do agree it's one method of such behavior but as fox said just removing the ears probably won't change much and it'll just affect the people that just think they're cute to use on their character.

**Comment by Ty-Omaha** [comment]
If people's characters _relies_ on having cat ears and a cat tail then I'm not sure that's a character that is beneficial to the RP environment.

**Comment by Darkmight9** [comment]
Ive also heard of discrimitory behavior directed at cat ear users such as being called 'cat' on coms. There was also that event a few days ago where cat ears were made illegal and I have no doubt that it made such players feel singled out and targeted. Things to consider

**Comment by Ty-Omaha** [comment]
>Ive also heard of discrimitory behavior directed at cat ear users such as being called 'cat' on coms. 

\>Wears cat ears & cat tail costume
\>Someone calls them a cat
\>shoked_pikachu.jpg

**Comment by Abydos2** [comment]
I have to agree with Shock, Ty and Sparks. The cat ears were originally added as a joke / meme item in SS13, that has since then spiraled out of control. People have since then, taken the item and surrounded their whole characters around them.. throwing a huge fit if the item is taken off their character for whatever reason, writing anime styled "uwu" descriptions as their examine text, to people literally claiming that they will die if done so and suiciding.

The ears within my opinion do not add anything to the environment, nor the server. From the standing infront of the bridge, spamming emotes, to the constant "Nya" at the end of every sentence, to people literally giving themselves jobs "Neko-(Insert job title here)", to people who cuff themselves (Or their friends) with the pink fluffy handcuffs, and buckling them to Ian's bed basket to be that persons "Pet" / "Cat".

It's cringy, it's not appropriate (Especially the last one) and very low roleplay.

**Comment by Darkmight9** [comment]
I very much do agree that this has all gotten out of had

**Comment by Darkmight9** [comment]
Just wanted to bring such things to attention

**Comment by Tayyyyyyy** [comment]
> Everyone is focusing on the memey and lewd cat ears users and yet no one is thinking about the people that just use them because it looks cute on their character.

There are the mouse ears which were a loadout item way before cat ears were added to loadout. They have the same tail wavey effect, yet I never saw a significant impact/complaints from them. It seems like even though there are other ways to low effort RP, these cat ears have become widely acknowledged as a uniquely big amplifier of low RP behavior. For those who just like it for being "cute", I don't see why the mouse ears don't function just as well. Sadly, this is another case of "this is why we can't have nice things".

I will say that if you like cat ears or RPing a cat ears character, there's nothing inherently wrong or cringey about it. It's just something whose novelty has worn off and can always be done in communities that are focused around anime, rather than in Paradise.

**Comment by Darkmight9** [comment]
Yeah this is sadly a case of "this is why we cant have nice things." Maybe mouse ears could be moved to to be a non donor loadout item to compensate for those who just want to wear cute ears for their characters.

**Comment by Carthusia** [comment]
> Yeah this is sadly a case of "this is why we cant have nice things." Maybe mouse ears could be moved to to be a non donor loadout item to compensate for those who just want to wear cute ears for their characters.

Heh, or move the cat ears to donator

**Comment by Necaladun** [comment]
Overall the pros seem to outweigh the cons here. I'd much prefer a station without people "roleplaying" as a cringey catgirl weeb. Having teenagers cosplaying as an anime meme is really not the intention for the roleplay levels on this server.

**Comment by Spacemanspark** [comment]
> Heh, or move the cat ears to donator

Inb4 people get the idea to donate to play an anime catgrille

**Comment by RaidarV** [comment]
Not a bad marketing idea tbh.

**Comment by Spacemanspark** [comment]
I'll make you a marketing idea

**Comment by Carthusia** [comment]
> > Heh, or move the cat ears to donator
> 
> Inb4 people get the idea to donate to play an anime catgrille

I mean....would that be so bad? If people REALLLY want it that bad, they will donate to get it, server gets upkeep money, overt weeb population drops.

**Comment by Spacemanspark** [comment]
Yes, it would be bad.
Why would we remove something that's been deemed as an issue to the server atmosphere only to lock it behind a pay wall.

That's fallout 76 levels of 'no' right there.

**Comment by Carthusia** [comment]
> That's fallout 76 levels of 'no' right there.

Nah, Fallout 76 levels of 'no' are restricting the clothing vendors and all loadout items not a jumpsuit behind a pay wall called "Space Bucks", while Ian is half stuck in a floor.

**Comment by Spacemanspark** [comment]
Our problems should not be 'solved' by paywalling in any form.

**Comment by Carthusia** [comment]
> Our problems should not be 'solved' by paywalling in any form.



> Our problems should not be 'solved' by paywalling in any form.

Fair enough, the last post was more an agreeable joke. Hence Ian

**Comment by BMane11** [comment]
Despite the cosplay argument for it to stay and whatnot, I mean... cat ears? Really. So removing them shouldn't be a big deal. And it might, just might, lessen the crude low-rp that does go on. 

Now while we are debating rp standards, I think there are bigger issues than cat ears. There's some very mary-sue like admin characters that I have seen in the past and in some cases, interacted with. I'm talking mainly about the demon ones. Some of their names escape me. There's that 'Sue' one, the child-like demon that frankly has no place on the server, is a complete mary-sue edgy character that just ends up being creepy and weird.

So while removing cat ears and such may or may not help the atmosphere, I think the discussion should be expanded to include other things, if the staff are serious about upping the quality of rp on the server. I also think it pays to lead by example, hence the statement about demon characters, etc.

**Comment by Spacemanspark** [comment]
Sue was removed ages ago along with most of the other demon characters.

**Comment by thatdanguy23** [comment]
> There's that 'Sue' one, the child-like demon that frankly has no place on the server, is a complete mary-sue edgy character that just ends up being creepy and weird.

That was the point. She was meant to be a horror character, and she actually contributed to a boost in RP during her events. 
In fact many of the demon characters were entirely RP-based, so their removal was pretty counterintuitive.

**Comment by Carthusia** [comment]
I kind of miss the demons. Added an element of mystery and that theres more out there than you know

**Comment by shawnhind** [comment]
Just my two cents but it seems like to me the goal of this PR is to discourage the anime girl style of RP.  I don't really have an opinion on whether this type of RP is bad or not.  I don't see that many people do it all that often and when I do see it I've never thought it to be cringy or overly creepy but maybe there's been some cases in the past where it has been over the top.  

However, if discouraging that behaviour is the goal, it seems to me like this PR isn't the right approach.  That goal would be better met by a rule or RP guidelines update, not a PR to remove a single cosmetic item which may or may not have any impact on the change that's actually desired.

I also don't think watering down RP options to make everyone act more "normal" is the correct approach either though necessarily.  I mean we have the Vox, a race where their main trait is to talk like children, even in command positions where it probably wouldn't make sense that they would be able to progress that high in the ranks without learning to speak proper Common (Voxxy has captain hats, yaya).  But this has also never bothered me and I actually like it because it's a fictional world and to some level requires some suspension of disbelief.

I personally don't care about the cat ears at all and it wouldn't have any impact on me if they do get removed.  But I do know that if there was a cosmetic item that my characters always used and it got removed because of what some people who use the item do, that would annoy me.  For that reason, I don't really like the precedent of removing items used by people who RP in a way that you don't like in the hopes that it makes them stop RPing that way.

**Comment by FreeStylaLT** [comment]
After fifty-one consecutive, insightful comments, I can guarantee you all that your feedback has been taken into account, one way or another. I suggest not stepping foot here again and leaving this place for the judgement of the _gods_.

**Comment by Pixelzilla** [comment]
"You're at a workplace, not an anime convention."
Also admins > spawning in poke'mon sprites during rounds
????

If the problem isn't specifically anime references and is specific player behaviour then why exactly dont admins speak with the individuals that are being problematic. Surely not everyone who uses cat ears is doing something deemed "cringe" on the basis of a single cat ear sprite alone. Isn't the purpose of having an admin team to deal with issues like this and not to punish the entire playerbase by removing an item that on its own is harmless?

Personally, I've seen a lot of people wearing the cat ears and not acting what we're deeming "bad". Maybe the "bad" few stick out so much because you're specifically looking for those sorts of players to base an argument on and overlooking the people who don't behave that way. I would challenge that its probably less than 10% of players doing the behaviour that is trying to be discouraged. Unless we're arguing that the very act of wearing cat ears means you're somehow the most low effort character in the history of characters.

Punish the behaviour. Punish the players responsible because they're probably going to move on to something else since the behaviour itself was never addressed.

**Comment by PidgeyThePirate** [comment]
Improve, don't remove.

What will solve this problem is a better set of cat ears.

**Comment by Spacemanspark** [comment]
No

**Comment by 81Denton** [comment]
merge conflict

**Comment by 81Denton** [comment]
wait wrong repo disregard that

**Comment by AffectedArc07** [comment]
@81Denton I mean you aint wrong

**Comment by Drakeven** [comment]
After a month of Cat ears still being in code, I'd like to point out that I, thus far, STILL have not heard/seen anyone wearing cat ears pretend to be a catgirl and use it as anything other than a cute accessory. Personally I think this PR is just an overblown reaction to a small issue/event that happened last month, and its instigation just serves to remove a well-loved accessory that I've seen many, many people use (Not just slime people, and not just people who are obsessed with being 'Neko girls', not that I've actually seen the latter.)

**Comment by thatdanguy23** [comment]
> other than as a cute accessory


yeah but just because these people who use it think it's "cute" and "endearing" doesn't mean it actually is
it's like players who spam "nya" at the end of their sentences

**Comment by Spacemanspark** [comment]
I think blowing up the station is endearing

**Comment by DashCee** [comment]
If you're going to remove this, then remove schoolgirl uniform
and maid outfit
and maid costume
and Victorian dresses
and skirts
and ...

The only reason you see cat ears on erp assistants so often is that it's one of the only head items in the game that isn't bland or masculine. Removing it isn't going to do anything, other than result in people doing the exact same thing, just without cat ears.

SS13 is fun because of seeing xenomorph janitors in maid outfits cleaning the station and bearded men in dresses doing surgery. Don't take that away.

**Comment by AffectedArc07** [comment]
`it's one of the only head items in the game that isn't bland or masculine.`

image
99% of these are unisex.

**Comment by Carthusia** [comment]
To be fair, the majority of those are not readily accessible, or in a chunk of them, not accessible at all to regular players and or without a hardsuit to go along with them.

And a chunk of them are not unirace, such as paper bags and other odd masks

**Comment by Spacemanspark** [comment]
If you personally believe that the game is interesting purely because of things such as maid skirt wearing xeno morphs (a joke from a fetish, which in of itself is in contention with staff as to whether or not it should be kept) and men in skirts, this might not be the server for you.

**Review by Kyep — APPROVED** [review]

**Comment by Kyep** [comment]
After hearing tales of how these get used, I'm voting for this.
It doesn't need my vote. It already has head approval. But I'm adding a second head approval to it.

**Comment by Spacemanspark** [comment]
Oh, I thought this died after the community vote said they didn't want them to be removed.

**Comment by RaidarV** [comment]
Idea, make cat ears a loadout item, but its locked behind 1500 hours of clown play time. So if we see anyone with cat ears, we can point and laugh at them whilst calling them clowns... Which they are.

**Comment by variableundefined** [comment]
This PR is more controversial than the rest of the Github flipping combined. 

I'm the biggest weeb on maintainer team so I'll vote yes.

**Comment by Mitchs98** [comment]
I'll still never understand why this was merged instead of handled by staff when issues arose with the item, but oh well.

**Comment by Tokorizo** [comment]
I have one word for this PR as a whole.
Lmao
Interpret as you will
